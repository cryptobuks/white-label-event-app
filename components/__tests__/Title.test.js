import React from 'react';
import renderer from 'react-test-renderer';
import Title from '../Title';

describe('Title', () => {
  it('renders correctly', () => {
    const comp = <Title value="titleValue" />;

    const tree = renderer.create(comp).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
