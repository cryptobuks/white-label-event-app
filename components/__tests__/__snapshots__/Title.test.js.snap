// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Object {
      "color": "black",
      "fontSize": 16,
      "fontWeight": "600",
    }
  }
>
  titleValue
</Text>
`;
